### ISSUE TITLE
trackpad not working

### ISSUE THREAD

**Issue by Parthvsquare** [ShiningXu/LG-Gram-macOS#12]
machine: z980 14inch, 
tried voodooI2C.kext but nothing works,
Downloaded image from official app store

**Comment by tyde7** [comment]
Just upgrade to VoodooI2C 2.3. My touchpad is ok now~ (not always, single point touch still not working)

**Comment by Parthvsquare** [comment]
Thanks mate, it's working now I can use single point but it's buggy, what kexts do you add

**Comment by tyde7** [comment]
com.alexandred.VoodooI2C (2.3) 
com.alexandred.VoodooI2CHID.service
com.alexandred.VoodooI2CHID
same situation, buggy touchpad. I'm using mouse now.

**Comment by LikeTiti** [comment]
@Parthvsquare @tyde7 

Hey there ! I'm facing the same issue with my14zd980. I tried to use this repository without success, it boots and I can use the keyboard, but the trackpad and touchscreen doesn't work. I tried to replace the kext as suggested by @tyde7 on my usb drive with the version 2.3, but it did not work either.

Did you guys do anything else to get it working ? Would you mind sharing your EFI folder ?

**Comment by Jkfengbo223** [comment]
> @Parthvsquare @tyde7
> 
> Hey there ! I'm facing the same issue with my14zd980. I tried to use this repository without success, it boots and I can use the keyboard, but the trackpad and touchscreen doesn't work. I tried to replace the kext as suggested by @tyde7 on my usb drive with the version 2.3, but it did not work either.
> 
> Did you guys do anything else to get it working ? Would you mind sharing your EFI folder ?


I did n’t respond to the touchpad at the beginning. Yesterday I tried to use the latest 2.4 version of VoodooI2. As a result, I almost solved the problem of my lg gram13z980 ’s touchpad. Now, the gesture function of single finger and multi finger is normal. , Friends who have problems with the touchpad can give it a try.
Solution: Mount the EFI partition with clover configuration, download version 2.4 of VoodooI2C-2.4.zip, and after decompression, simply place the two files VoodooI2C.kext and VoodooI2CHID.kext in the package into the EFI / clover / kext / other / path , Replace the original old version of VoodooI2C.kext and VoodooI2CHID.kext, and then restart the notebook, the touchpad can be used. My 10.15.3. System, problem solved successfully.
Here is the download link of VoodooI2C-2.4.zip:
https://github.com/alexandred/VoodooI2C/releases
